mrc/shared/relativeposition.py

```diff
@@ -0,0 +1,5 @@
+class RelativePosition:
+    def __init__(self, radius: float, angle: float, bot_to_follow):
+        self.bot_to_follow = bot_to_follow
```

Comment:
I don't think that bot_to_follow belongs here. If we store positions in map[robot_id -> relative_position] then it's redundant.

Reply:
Good point